projects/plugins/protect/package.json

```diff
@@ -25,6 +25,7 @@
 		"extends @wordpress/browserslist-config"
 	],
 	"dependencies": {
+		"@automattic/jetpack-api": "workspace:* || ^0.13",
```

Comment:
The needed dependency is `@wordpress/api-fetch` (probably at 6.3.0 since that's what we already have elsewhere), not `@automattic/jetpack-api`.

Reply:
oops